mandatory/2-function-creation.js

```diff
@@ -15,26 +17,56 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+return totalPrice * 0.9;
```

Comment:
having proper indenting inside functions and blocks is usually a good way to make sure code is readable. You should make sure to use an editor that has auto-indenting enabled as that helps making sure everything is in order

Reply:
i'm using vsc, is there any extensions i can use for this?